client/state/reader/streams/selectors/get-reader-stream-transformed-items.js

```diff
@@ -26,6 +31,8 @@ export const getTransformedStreamItems = treeSelect(
 			items = injectRecommendations( items, recs, getDistanceBetweenRecs( follows.length ) );
 		}
 
+		items = injectPrompts( items, getDistanceBetweenPrompts( follows.length ) );
```

Comment:
Nice that this hook already exists for this 👍

Reply:
Agree! Easier to reuse  existing pattern